spec/vttutils.spec.ts

```diff
@@ -0,0 +1,572 @@
+import { SubtitleRegionContainer, SubtitleLabel } from '../src/ts/components/subtitleoverlay';
+import { VttUtils } from '../src/ts/vttutils';
+import { VTTRegionProperties, VTTProperties } from 'bitmovin-player';
+import { MockHelper } from './helper/MockHelper';
+
+describe('Vtt Utils', () => {
+  describe('Vtt Region', () => {
+    it('should set region css properties', () => {
+        const mockRegionContainer = generateSubtitleRegionContainerMock();
+        const mockDomElement = MockHelper.generateDOMMock();
+        mockRegionContainer.getDomElement = () => mockDomElement;
+        const spyCss = jest.spyOn(mockRegionContainer.getDomElement(), 'css');
+    
+        VttUtils.setVttRegionStyles(mockRegionContainer, vttRegionProps, {width: 1000, height: 800});
+
+        expect(spyCss).toHaveBeenCalledTimes(8);
+        expect(spyCss).toHaveBeenNthCalledWith(1, 'position', 'absolute');
+        expect(spyCss).toHaveBeenNthCalledWith(2, 'overflow', 'hidden');
+        expect(spyCss).toHaveBeenNthCalledWith(3, 'width', `${vttRegionProps.width}%`);
+        expect(spyCss).toHaveBeenNthCalledWith(4, 'left', '75px');
+        expect(spyCss).toHaveBeenNthCalledWith(5, 'right', 'unset');
+        expect(spyCss).toHaveBeenNthCalledWith(6, 'top', '772px');
+        expect(spyCss).toHaveBeenNthCalledWith(7, 'bottom', 'unset');
+        expect(spyCss).toHaveBeenNthCalledWith(8, 'height', '28px');
+      });
+  });
+
+  describe('Vtt Cue Box', () => {
+    it('should set text align', () => {
+      const mockRegionContainer = generateSubtitleCueBoxMock(false, { align: 'left' } as VTTProperties);
+      const mockDomElement = MockHelper.generateDOMMock();
+      mockRegionContainer.getDomElement = () => mockDomElement;
+      const spyCss = jest.spyOn(mockRegionContainer.getDomElement(), 'css');
+
+      VttUtils.setVttCueBoxStyles(mockRegionContainer);
+      
+      expect(spyCss).toHaveBeenCalledTimes(11);
+      expect(spyCss).toHaveBeenNthCalledWith(8, 'text-align', 'left');
+    });
+
+    it('should not set default position', () => {
+      const mockRegionContainer = generateSubtitleCueBoxMock(false, { position: 50 } as VTTProperties);
```

Comment:
Shouldn't that be `auto` now instead of `50`?

Reply:
The test says "it should NOT set default" 😄